src/test/scala/eventstore/StreamSourceSpec.scala

```diff
@@ -290,30 +291,6 @@ class StreamSourceSpec extends SourceSpec {
       override def eventNumber = Some(EventNumber(0))
     }
 
-    "resubscribe from same position" in new SourceScope {
-      connection expectMsg readEvents(0)
-      connection reply readCompleted(0, endOfStream = true)
-      connection expectMsg subscribeTo
-      connection reply subscribeCompleted(0)
-      expectNoActivity()
-
-      override def eventNumber = Some(EventNumber(0))
-    }
-
-    "resubscribe from different position" in new SourceScope {
```

Comment:
This one is redunant, bacause of `handle unexpected resubscribe while subscribed` and `handle unexpected resubscribe while catching up`